osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -136,7 +138,10 @@ private void updateSpinningSample(ValueChangedEvent<bool> tracking)
             }
             else
             {
-                spinningSample?.VolumeTo(0, 300).OnComplete(_ => spinningSample.Stop());
+                if (spinningSample != null)
+                    spinningSample.Volume.Value = 0;
+
+                spinningSample?.VolumeTo(0, fade_out_duration);
```

Comment:
I'm a bit confused. So it's an instantaneous fade to 0... followed by a transform? Is this right?

Reply:
The set to zero was a debug line; unintentional.